build.gradle

```diff
@@ -426,7 +426,8 @@ project(':ambry-cloud') {
             project(':ambry-utils'),
             project(':ambry-commons'),
             project(':ambry-account'),
-            project(':ambry-replication')
+            project(':ambry-replication'),
+            project(':ambry-server')
```

Comment:
We should not introduce a dependency on `ambry-server` for `ambry-cloud`. This has a potential of bringing a lot of cyclical dependencies in future. Also this will bloat the jars will a lot of unnecessary libraries.

What we can do here is move `StorageServerNettyFactory` and `StorageServerNettyChannelInitializer` to `ambry-rest`. Maybe rename them as `StorageNettyFactory` and `StorageNettyChannelInitializer`. And then import `ambry-rest` instead of `ambry-server`, and also import `ambry-network` in `ambry-rest` project.

Reply:
Comment addressed.